### ISSUE TITLE
Scale

### ISSUE THREAD

**Issue by jodevak** [jesperborgstrup/Py-IBLT#1]
Thank you for your implementation, i tried to make insert 1000000 pairs with a design threshold of 1500000 , but its taking forever when i try to list entries

**Comment by jesperborgstrup** [comment]
Yes I'm not surprised. The code is not optimized for performance, but rather for readability. I'll gladly merge any PR that improves performance without making it completely unreadable :-)

**Comment by justincpresley** [comment]
A faster Py-IBLT would be very useful for network applications I deal with/am developing. If someone has time to work on it, one quick performance boost would be to use the fastest bit array implementation which would be **_gmpy2.xmpz_** or possibly **_intbitset_** (depending on recent changes) tested [here](https://stackoverflow.com/questions/20845686/python-bit-array-performant).
